index.html

```diff
@@ -222,5 +222,35 @@ <h2>Multi-Post Stories</h2>
     <!-- SEE PROJECT BUTTON -->
     <button class="btn">See project</button>
     </div>
+
+    <section class="contact-form">
+   
+      <h2 class="contact-us">Contact Us</h2>
+      <p class="paragraph3">If you have an application you are
+        interested in developing, a feature that you
+        need built or a project that needs coding.
+        I'd love to help with it </p>
+    
+      <form action="https://formspree.io/forms/meqdknwb/integration" method="post" class="form">
+      <input type="text" id="name" name="name" placeholder="User name" maxlength="30" required>
```

Comment:
- [ ] Kindly replace the User name with a first name or last name

Reply:
ok , i will , thank you sir.